adb-vagrantfile-tests/adb-release-mesos/test-new-build-mesos.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+import os
+import unittest
+import subprocess
+
+from unittest import TestCase
+
+VAGRANTFILE_PATH = \
+    "/root/adb/components/centos/centos-mesos-marathon-singlenode-setup/" \
+    "Vagrantfile"
+
+CWD = "/root/adb/components/centos/centos-mesos-marathon-singlenode-setup/"
```

Comment:
Other option is to change directories within the code. And that would make things confusing when the code is read later or by someone else. Using `cwd=` option helps execute command from a particular directory without one having to keep a track of directory paths.